packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/PendingRecordingProxyApi.java

```diff
@@ -25,6 +27,19 @@ public ProxyApiRegistrar getPigeonRegistrar() {
     return (ProxyApiRegistrar) super.getPigeonRegistrar();
   }
 
+  @NonNull
+  @Override
+  public PendingRecording withAudioEnabled(PendingRecording pigeonInstance, boolean initialMuted) {
```

Comment:
Just to clarify, `withAudioEnabled(true)` disables audio and `withAudioEnabled(false)` enables audio? I'm confused by the parameter name being `initialMuted`.

Reply:
Lol yeah it's confusing.

I made this mirror [`PendingRecording.withAudioEnabled`](https://developer.android.com/reference/androidx/camera/video/PendingRecording#withAudioEnabled(kotlin.Boolean)) where `initialMuted == true` means that you want to initialize the recording as being muted/audio disabled, so `withAudioEnabled(true)` disables audio, `withAudioEnabled(false)` enables audio.